plugins/woocommerce/client/legacy/css/forms.scss

```
@@ -0,0 +1,226 @@
+.woocommerce {
+	/**
+	 * Generic forms styles used in places such as my account and the shortcode based checkout.
+	 */
+	form .form-row {
+		padding: 3px;
+		margin: 0 0 6px;
+
+		[placeholder]:focus::-webkit-input-placeholder {
+			transition: opacity 0.5s 0.5s ease;
+			opacity: 0;
+		}
+
+		label {
+			line-height: 2;
+		}
+
+		label.hidden {
+			visibility: hidden;
+		}
+
+		label.inline {
+			display: inline;
+		}
```

Nitpic: As we're using SCSS, we could optimise this part slightly:
```suggestion
		label {
			line-height: 2;

			&.hidden {
				visibility: hidden;
		  	}

			&.inline {
				display: inline;
			}
		}
```